src/Internal/BalanceTx/Error.purs

```diff
@@ -89,6 +90,37 @@ instance Show BalanceTxError where
     "ExUnitsEvaluationFailed: " <> printTxEvaluationFailure tx failure
   show e = genericShow e
 
+explainBalanceTxError :: BalanceTxError -> String
+explainBalanceTxError = case _ of
+  BalanceInsufficientError expected actual ctx ->
+    "Insufficient balance. Expected: " <> show expected
+      <> ", actual: "
+      <> show actual
```

Comment:
I get an error of this type when trying `pprintValue` here:

```
[1/1 TypesDoNotUnify] src/Internal/BalanceTx/Error.purs:97:62

  97      "Insufficient balance. Expected: " <> show (pprintValue (unwrap expected))
                                                                   ^^^^^^^^^^^^^^^
  
  Could not match type
  
    Value
  
  with type
  
    Value
  
  while solving type class constraint
  
    Data.Newtype.Newtype Expected
                         Value
  
  while applying a function unwrap
    of type Newtype t0 t1 => t0 -> t1
    to argument expected
  while checking that expression unwrap expected
    has type Value
  in value declaration explainBalanceTxError
  
  where t0 is an unknown type
        t1 is an unknown type
```

Is it because the `Cardano.Types` `Value` is different to the `Plutus.Types` `Value`?

Reply:
Then please use `Ctl.Internal.Plutus.Conversion.Value` functions for conversion before pretty-printing.